src/DataTable/DataTable.svelte

```diff
@@ -132,7 +132,7 @@
       .split(/[\.\[\]\'\"]/)
       .filter((p) => p)
       .reduce(
-        (o, p) => (o && typeof o === "object" && o[p] ? o[p] : defaultValue),
+        (o, p) => ((o && typeof o === "object") ? o[p] : defaultValue),
```

Comment:
I'm not sure if this solves the issue. Having it fallback to the defaultValue would result in the empty string.